src/abstractdataframe/io.jl

```diff
@@ -178,7 +178,7 @@ end
         write(io, "<th>$column_name</th>")
     end
     write(io, "</tr>")
-    tty_rows, tty_cols = Base.displaysize(io)
+    tty_rows, tty_cols = Compat.displaysize()
```

Comment:
No, you need to pass `io` (at least where possible), as it is the recommended way in Julia 0.5.